src/main/java/decodes/tsdb/algo/HdbEvaporation.java

```diff
@@ -1,61 +1,38 @@
 package decodes.tsdb.algo;
 
-import java.util.Date;
-
-import ilex.var.NamedVariableList;
 import ilex.var.NamedVariable;
-import decodes.tsdb.DbAlgorithmExecutive;
 import decodes.tsdb.DbCompException;
-import decodes.tsdb.DbIoException;
-import decodes.tsdb.VarFlags;
-
-//AW:IMPORTS
-import decodes.tsdb.RatingStatus;
-import decodes.hdb.HDBRatingTable;
-//AW:IMPORTS_END
-
-//AW:JAVADOC
-/**
-Implements an evaporation computation
-Primary value is "area".
-Really need average area, so two choices for second input:
-prevArea = previous EOP area or diffArea = deltaArea
-For now, chose prevArea.
-Third input is evapCoeff. This is a SDI pointer to either
-a coefficient timeseries, or to lookup a coefficient from the stat tables
-Output is evaporation as a volume.
-<p>Properties include: 
-<ul> 
-<li>ignoreTimeSeries - completely ignore changes to any timeseries value from evapCoeff, and always lookup from database.
-</li>
-</ul>
-
-
- */
-//AW:JAVADOC_END
+import org.opendcs.annotations.algorithm.Algorithm;
+import org.opendcs.annotations.algorithm.Input;
+import org.opendcs.annotations.algorithm.Output;
+
+@Algorithm(description = "Implements an evaporation computation\n" +
+		"Primary value is \"area\".\n" +
+		"Really need average area, so two choices for second input:\n" +
+		"prevArea = previous EOP area or diffArea = deltaArea\n" +
+		"For now, chose prevArea.\n" +
+		"Third input is evapCoeff. This is a SDI pointer to either\n" +
+		"a coefficient timeseries, or to lookup a coefficient from the stat tables\n" +
+		"Output is evaporation as a volume.\n" +
+		"Properties include: \n" +
+		"\n" +
+		"ignoreTimeSeries - completely ignore changes to any timeseries value from evapCoeff, and always lookup from database.\n" +
```

Comment:
I removed the Html markup previously used for Javadoc's, but not inside annotation;  I don't think the text box will render Html?

Reply:
We could keep javadocs also, but that feels like duplication.

Reply:
We could also make that box render HTML; Swing has some support for the basics. But we'd need some way identify that it is HTML.

Reply:
I'll keep the HTML for future rendering if HTML is detected, I don't think Html will hurt in text rendering.  Many algos only have text in the javadocs